lib/suspenders/generators/app_generator.rb

```diff
@@ -54,7 +54,12 @@ def suspenders_customization
     end
 
     def customize_gemfile
-      build :replace_gemfile
+      if 'mongodb' == options[:database]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.